src/main.rs

```diff
@@ -110,157 +104,180 @@ fn main() {
 	.expect("Something went wrong :( Could not read the file");
 
     let word_hashset = assemble_word_hashset(&word_file_contents);
-    let cn_word_hashset = assemble_word_hashset(&cn_file_contents);
 	let ranked_words_hashset = assemble_suggestion_hashset(&ranked_words_contents);
 
-    search(&contents, word_hashset, cn_word_hashset, ranked_words_hashset);
+    search(&contents, word_hashset, ranked_words_hashset);
 
     let end = PreciseTime::now();
 	let time_taken = format!("{}", start.to(end));
 	let time_taken = &time_taken[2..];
     println!("Took {} seconds to spell-check.", time_taken);
 }
 
-pub fn search<'a>(contents: &'a str, word_hashset :  HashSet<&'a str>, cn_word_hashset : HashSet<&'a str>, 
-					ranked_words_hashset: HashSet<&'a str>) {
-    let mut line_number = 0;
+pub fn search<'a>(contents: &'a str, word_hashset :  HashSet<&'a str>, ranked_words_hashset: HashSet<&'a str>) {
     let mut total_spelling_errors = 0;
     let mut word_count = 0;
-
-    let sentences = contents.split(".");
+    
+    
+    let mut modified_contents = contents.replace("?", ".");
+    modified_contents = modified_contents.replace("!", ".");
+
+    modified_contents = modified_contents.replace(",", "");
+    modified_contents = modified_contents.replace(";", "");
+    modified_contents = modified_contents.replace(":", "");
+    modified_contents = modified_contents.replace("[", "");
+    modified_contents = modified_contents.replace("]", "");
+    modified_contents = modified_contents.replace("(", "");
+    modified_contents = modified_contents.replace(")", "");
+    modified_contents = modified_contents.replace("{", "");
+    modified_contents = modified_contents.replace("}", "");
+    modified_contents = modified_contents.replace("\"", "");
+
+
+    let modified_contents = modified_contents.split(".").collect::<Vec<&str>>();
+    
 
     let mut sentences_and_questions = Vec::new();
-    
-    for sentence in sentences {
-        if sentence.contains("?") {
-            let questions = sentence.split("?");
-            for q in questions {
-                sentences_and_questions.push(q);
-            }
-        } else {
-            sentences_and_questions.push(sentence);
-        }
-    }
 
-    for i in &sentences_and_questions {
-        println!("{}", i);
+    for sentence in modified_contents {
+        if !sentence.is_empty() {
```

Comment:
Cool way if checking if there's content in the sentence . . . Ya learn something new every day XD